src/NHibernate/Driver/SqlClientDriver.cs

```diff
@@ -12,23 +13,49 @@ namespace NHibernate.Driver
 	/// <summary>
 	/// A NHibernate Driver for using the SqlClient DataProvider
 	/// </summary>
+#if DRIVER_PACKAGE
+	public class SqlServer2000Driver : DriverBase, IEmbeddedBatcherFactoryProvider
+#else
+	[Obsolete("Use NHibernate.Driver.SqlServer NuGet package and SqlServer2000Driver."
+		+ "  There are also Loquacious configuration points: .Connection.BySqlServer2000Driver() and .DataBaseIntegration(x => x.SqlServer2000Driver()).")]
 	public class SqlClientDriver : DriverBase, IEmbeddedBatcherFactoryProvider
```

Comment:
For this PR to be a complete preparation to .NetStandard, shouldn't this driver be switched to reflection based when compiled inside NHibernate.csproj?

Reply:
My intention was for .NetStandard to not have these reflection-based drivers at all.  If an application is migrating to .NetStandard, it isn't break-free, so no reason to introduce an already obsolete driver when it could be migrated ahead of time or at the same time.

Reply:
So the plan is to have those files excluded in csproj when target is .NetStandard? Ok.